core/src/test/scala/unit/kafka/cluster/PartitionTest.scala

```diff
@@ -1936,6 +1937,106 @@ class PartitionTest extends AbstractPartitionTest {
     callback(brokerId, remoteBrokerId, partition)
   }
 
+  def createClientResponseWithAlterPartitionResponse(topicPartition: TopicPartition,
+                                                     partitionErrorCode: Short,
+                                                     isr: List[Int] = List.empty,
+                                                     leaderEpoch: Int = 0,
+                                                     partitionEpoch: Int = 0
+                                                    ): ClientResponse = {
+    val alterPartitionResponseData = new AlterPartitionResponseData()
+    val topicResponse = new AlterPartitionResponseData.TopicData()
+      .setTopicName(topicPartition.topic())
+
+    topicResponse.partitions.add(new AlterPartitionResponseData.PartitionData()
+      .setPartitionIndex(topicPartition.partition())
```

Comment:
nit: There are a few places where we could omit the parenthesis when calling getters.